build-system/tasks/pr-check.js

```diff
@@ -93,10 +89,19 @@ async function prCheck() {
 
   if (buildTargetsInclude(Targets.RUNTIME)) {
     runCheck('amp dep-check');
-    runCheck('amp check-types');
     runCheck('amp check-sourcemaps');
   }
 
+  if (
+    buildTargetsInclude(
+      Targets.RUNTIME,
+      Targets.BUILD_SYSTEM,
+      Targets.EXTENSIONS
```

Comment:
Drive-by comment: This only takes care of the logic for the `amp pr-check` task. We'll also need to update every other spot that includes `Targets.RUNTIME` to also include `Targets.EXTENSIONS`, without which CI will stop running tests for PRs that modify extensions.